CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ include_directories(src)
 
 file(GLOB_RECURSE SOURCES src/**.cpp src/**.hpp)
 
-add_executable(openfusion ${SOURCES})
+add_executable(openfusion ${SOURCES} "src/Database.cpp" "src/contrib/sqlite/shell.c" "src/contrib/sqlite/sqlite3.c" "src/contrib/sqlite/sqlite3pp.cpp" "src/contrib/sqlite/sqlite3ppext.cpp" "src/contrib/bcrypt/bcrypt.c" "src/contrib/bcrypt/crypt_blowfish.c" "src/contrib/bcrypt/crypt_gensalt.c" "src/contrib/bcrypt/wrapper.c")
```

Comment:
Also, you have to remove `src/Database.cpp`, `src/contrib/sqlite/sqlite3pp.cpp`, and `src/contrib/sqlite/sqlite3ppext.cpp` from the list.

Reply:
@Raymonf would the following work:
```cmake
file(GLOB_RECURSE SOURCES src/**.c* src/**.h*)

add_executable(openfusion ${SOURCES})
```
If it does, this might be a more generic solution for the long run.

Reply:
I would honestly much prefer `file(GLOB_RECURSE SOURCES src/**.cpp src/**.c src/**.hpp src/**.h)`, but maybe.

Reply:
Yeah that's better. I think it would save us a lot of hassle in case we need new source-based libraries.